site_scons/site_init.py

```diff
@@ -129,7 +129,7 @@ def BuildEnvironmentSConscripts(env):
     else:
       raise SCons.Errors.UserError(
           'Bad location for a SConscript. "%s" is not under '
-          '\$TARGET_ROOT or \$MAIN_DIR' % c_script)
+          r'\$TARGET_ROOT or \$MAIN_DIR' % c_script)
```

Comment:
I believe in this one, the backslashes are actually not wanted in the output and should be removed instead.